public/course_wizard.php

```diff
@@ -0,0 +1,261 @@
+<!--This works for now but should probably be refactored into separate html/php files.
+It was mostly to try mixed php/html and get something working with basic styling
+and data entry. -->
+
+<html>
+
+    <?php
+
+    require "../modules/models/course.php";
+    require "../modules/models/course_offering.php";
+    require "../modules/models/course_section.php";
+    require_once "../common.php";
+
+    try  {
+        $result_courses = Course::getAll();
+        $result_offerings = CourseOffering::getAll();
+        $result_sections = CourseSection::getAll();
+    } catch(PDOException $error) {
+        echo $sql . "<br>" . $error->getMessage();
```

Comment:
$sql is not defined here

Reply:
I corrected this by removing the $sdl and replacing it with a regular string for the error followed by getMessage().